src/videojs-contrib-hls.js

```diff
@@ -95,6 +95,25 @@ export default class HlsHandler extends Component {
     this.bandwidth = this.options_.bandwidth || 4194304;
     this.bytesReceived = 0;
 
+    // listen for fullscreenchange events for this player so that we
+    // can adjust our quality selection quickly
+    if (document.fullscreenEnabled || document.webkitFullscreenEnabled) {
+      this.on(document, [
```

Comment:
this is because the `fullscreenchange` bubbles up from the player element to the document and as a way to get around the infinite loop that could be created by listening to it on the player/tech?